sysbench/run.bash.ticat

```diff
@@ -0,0 +1,36 @@
+help = run TPCC benchmark
+
+[arg]
+tables|table|tb = ''
+table-size|size = ''
+test-name|name = oltp_point_select
+threads|thread|thrd|thd|t|T = ''
+duration|dur|d|D = ''
+host|h|H = ''
+port|p|P = ''
+user|u|U = root
+
+[arg2env]
+bench.sysbench.threads = threads
+bench.sysbench.tables = tables
+bench.sysbench.table-size = table-size
+bench.sysbench.test-name = test-name
+bench.sysbench.duration = duration
+mysql.host = host
+mysql.port = port
+mysql.user = user
+
+[env]
+bench.sysbench.threads|thread|thrd|terminal|term|t|T = read
+bench.sysbench.tables|table|tb = read
+bench.sysbench.table-size|size = read
+bench.sysbench.test-name|name = read
+bench.sysbench.duration|dur|d|D|time = read
+mysql.host = read
+mysql.port = read
+mysql.user = read
+bench.run.log = write
+bench.run.score = write
+
+[dep]
+sysbench = to run the bench
```

Comment:
Consider this: add sysbench as git-sub-module under <this-repo>/repos, then built it if not have sysbench installed.
(have did that in the old version tool `ti.sh`, it runs better with auto installing)

Reply:
Or not adding sysbench as git-sub-module, only pull it down to a temp dir when it's needed

Reply:
We may currently assume the users are on popular distributions and have a known package manager installed, then we can easily install the missing dependencies from it.